### DISCUSSION TITLE
ATTN!

### DISCUSSION THREAD

**Discussion by tatianatt7** [community/community#64255] [Programming Help]
### Body

Good night , dear colleagues,

I accidentally deleted my repository https://github.com/tatianatt7/react-mesto-api-full-gha. can you restore it? Deadline is coming on my courses.The question is very urgent!=) 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by lab596** [comment]
Hey, here is the step-by-step: 

1. Click on your profile in the upper right corner
2. Go to `Settings`
3. Go to `Repositories`  under ****Code, planning, and automation****
4. Click on `Deleted Repositories` above your profile name
5. Recover said Repository

For additional documentation and picture guidance click [here](https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository)
